src/display.js

```diff
@@ -0,0 +1,42 @@
+class Display {
+  display = () => {
+    /* tasks list */
+    const tasks = [
+      {
+        index: 1,
+        description: 'structure todo list',
+        completed: 'false',
+      },
+      {
+        index: 2,
+        description: 'add task todo list',
+        completed: 'false',
+      },
+      {
+        index: 3,
+        description: 'remove todo list',
+        completed: 'false',
+      },
+    ];
+
+    const todoDiv = document.getElementById('todoDiv');
+    const caption = document.createElement('p');
+    caption.innerText = "Today Todod's";
+    /*  todoDiv.appendChild(caption); */
+    /* loop through tasks */
+    let str = "   Todays todo's";
+    tasks.map((singleTask) => {
+      str += `
+      <ul class="todolistItem">
+      <li><i class="icon-check-empty"></i>${singleTask.index}</li>
+      <li>${singleTask.description}</li>
+      <li>${singleTask.completed}</li>
+      </ul>
+      `;
+      return str;
+    });
+    todoDiv.innerHTML = str;
```

Comment:
- [ ] Kindly make sure that your design matches the provided wireframe. You have done a great effort in listing the individual tasks. Now let's make the UI  look like the provided wireframe.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/185333873-06e5ff10-815b-4e27-858b-1ab931b6a1bb.png)

**This is the desired design as provided by the wireframe:**

![image](https://user-images.githubusercontent.com/86318284/185333408-64c2bc99-95ae-4f6f-897a-8673956e0ddc.png)

Reply:
- Hello @Nemwel-Boniface 
- In Activity requirements, it is specified styling is optional, so I left the styling for now.
- As todo list styling is required for the next activity, definitely i will be implementing the design